### PR TITLE
feat: add missing mtp filed

### PR DESCRIPTION
**Pull request by politeWall**

### CODE REVIEW

File: bindings-test/src/tests.rs [hunk 1]
```diff
@@ -126,6 +126,12 @@ fn asset_info_not_found() {
 fn query_positions() {
     let mtps: Vec<Mtp> = vec![Mtp {
         address: "user".to_string(),
+        collateral_asset: "".to_string(),
+        trading_asset: "".to_string(),
+        liabilities_asset: "".to_string(),
+        custody_asset: "".to_string(),
+        take_profit_borrow_rate: Decimal::zero(),
+        open_price: Decimal::zero(),
         collaterals: vec![],
```
Comment (cosmic-vagabond) [inline bindings-test/src/tests.rs:135]: @politeWall this field does not exist anymone now it is a single `collateral` please check all the fields and make sure theey are aligned with the native module MTP structure

**Review by cosmic-vagabond — CHANGES_REQUESTED** [review]

File: bindings-test/src/tests.rs (hunk 1 above)
Comment (politeWall) [reply bindings-test/src/tests.rs:135]: i will do that right away

File: bindings-test/src/multitest.rs [hunk 2]
```diff
@@ -400,46 +400,52 @@ impl Module for ElysModule {
...
+                let mut mtp_vec = MARGIN_OPENED_POSITION.load(storage)?;
 
-                let order_id: u64 = match order_vec.iter().max_by_key(|s| s.id) {
-                    Some(x) => x.id + 1,
+                let mtp_id: u64 = match mtp_vec.iter().max_by_key(|mtp| mtp.id) {
+                    Some(mtp) => mtp.id + 1,
                     None => 0,
                 };
-                let collaterals = vec![collateral];
+                let collaterals = vec![collateral.clone()];
```
Comment (cosmic-vagabond) [inline bindings-test/src/multitest.rs:409]: @politeWall `collateral` is not supposed to be a vector

File: bindings-test/src/multitest.rs [hunk 3]
```diff
@@ -400,46 +400,52 @@ impl Module for ElysModule {
...
 
                 let resp = AppResponse {
                     events: vec![],
                     data: Some(to_json_binary(&msg_resp)?),
                 };
 
-                order_vec.push(order);
+                mtp_vec.push(mtp);
 
                 let burn_msg = BankMsg::Burn {
```
Comment (cosmic-vagabond) [inline bindings-test/src/multitest.rs:450]: @politeWall and why do you call `bankmsg::burn` ?

**Review by cosmic-vagabond — APPROVED** [review]

**Pull request merged**